app/views/teams/_individuals_table.html.erb

```diff
@@ -11,7 +11,7 @@
               <%= User.human_attribute_name('name') %>
             </th>
             <th>
-              <%= Plan.model_name.human %>
+              <%= Plan.human_attribute_name('average_weekly_planned') %>
```

Comment:
Just looking at this in the UI, I think we should make the headings here match the text in the boxes on the individual data page i.e. Average planned per week, Average actual per week, Average percentage per week

![Screenshot 2021-03-02 at 09 05 36](https://user-images.githubusercontent.com/40230316/109624853-913bd500-7b36-11eb-9024-9c01db5af83e.png)

![Screenshot 2021-03-02 at 09 05 45](https://user-images.githubusercontent.com/40230316/109624872-96991f80-7b36-11eb-8dff-e2be894be073.png)

Reply:
makes sense, I will use the same locale path for both, so we can easy change it in both places